src/nl/documentatie/14.shopify.md

```diff
@@ -52,60 +43,93 @@ Om de MyParcel-plugin te installeren, volg je de volgende stappen:
 
 De app wordt bij wijzigingen automatisch bijgewerkt, je hebt hier zelf geen onderhoud aan.
 
+### Eenmalige handeling
+
+<MPImg src="/documentation/shopify/nl/shopify-api-key.jpg" alt="Shopify MyParcel API setting" />
+
+Om de MyParcel App in gebruik te nemen, moet er eerst een API key worden ingevuld. De API key is gratis te verkrijgen via <a href="https://backoffice.myparcel.nl/registration" target="_blank"> Shopinstellingen > API instellingen in de MyParcel Backoffice</a>.
+De MyParcel app wordt gekoppeld aan jouw Shopify shop wanneer de API key is ingevuld.
+
 ## Configuratie
 
 ### Algemene instellingen
 
-<MPImg src="/documentation/shopify/shopify-general-settings.jpg" alt="Shopify general settings" />
+<MPImg src="/documentation/shopify/nl/shopify-general-settings.jpg" alt="Shopify general settings" />
 
 #### Standaard paginakeuze
 
 Hier geef je aan of je het label op een standaard (A4) of op een labelprinter (A6) wilt printen.
 
 #### Beschrijving op label
 
-Met deze optie kun je een beschrijving toevoegen aan de zending. Dit zal worden afgedrukt op de linkerbovenhoek van het label en kun je gebruiken om de zending in het MyParcel zendingenoverzicht op te zoeken. Gebruik `{order_name}` om de titel van de order of `{order_number}` om het ordernummer te vermelden.
-
 <MPImg src="/documentation/shopify/shopify-label-description.jpg" alt="Shopify label description" />
 
+Met deze optie is het mogelijk een beschrijving toevoegen aan de zending. Dit zal worden afgedrukt op de linkerbovenhoek van het label en hiermee is het mogelijk om de zending in het MyParcel zendingenoverzicht op te zoeken. Gebruik `{order_name}` om de titel van de order of `{order_number}` om het ordernummer te vermelden.
+
 #### Fulfillment
 
-Je hebt de mogelijkheid om jouw orders automatisch de status 'Fulfilled' te geven, wanneer PostNL jouw zending heeft gescand. Wil je dit liever handmatig doen? Selecteer dan de optie Manueel. Ga naar Bestellingen, selecteer de gewenste orders en klik bovenaan het orderoverzicht op Bestellingen vervullen om deze status te veranderen.
+<MPImg src="/documentation/shopify/nl/shopify-fulfillment.jpg" alt="Shopify fulfillment" />
 
-### Account
+- Er is een optie om de bestellingen manueel / handmatig de status 'fulfilled' te geven. Ga naar Bestellingen, selecteer de gewenste orders en klik bovenaan het orderoverzicht op Bestellingen vervullen om deze status te veranderen.
+- Je hebt de mogelijkheid om jouw orders automatisch de status 'Fulfilled' te geven, wanneer PostNL jouw zending heeft gescand.
```

Comment:
Hier staat 'fulfilled' stelselmatig tussen enkele aanhalingstekens, maar verderop worden dubbele aanhalingstekens gebruikt voor letterlijke termen. Ik zou het in ieder geval binnen 1 handleiding consequent houden, en misschien over heel MyParcel zelfs.